docs/conf.py

```diff
@@ -91,6 +91,7 @@
         ("Examples", "auto_examples/index"),
         ("API", "api"),
         ("What's new", "whats_new"),
+        ("Differences from PREP", "matlab_differences"),
         ("GitHub", "https://github.com/sappelhoff/pyprep", True),
     ],
 }
```

Comment:
In the CI build of the docs you also find the following note:

> intersphinx inventory has moved: https://www.numpy.org/devdocs/objects.inv -> https://numpy.org/devdocs/objects.inv

a couple of lines below this one in the conf.py file, you find the line:

`    "numpy": ("https://www.numpy.org/devdocs", None),`

Can you adjust that to:

`    "numpy": ("https://numpy.org/devdocs", None),`

please? (I cannot comment there directly due to GitHub restrictions on which lines can be commented on (I think only +-5 lines next to a touched line)

Reply:
Done!